tests/10-issue.js

```diff
@@ -64,206 +69,248 @@ describe('Issuers - BitstringStatusList',
+                statusEntries[statusEntry].statusSize.should.be.gt(0,
+                  'Expected statusSize to be greater than zero.');
+              } else {
+                this.skip();
+              }
+            }
+          });
+          it('If statusSize is provided and is greater than 1, ' +
+            'then the property credentialStatus.statusMessage ' +
+            'MUST be present.',
+          async function() {
+            this.test.link = 'https://www.w3.org/TR/vc-bitstring-status-list/#:~:text=If%20statusSize%20is%20provided%20and%20is%20greater%20than%201%2C%20then%20the%20property%20credentialStatus.statusMessage%20MUST%20be%20present';
+            for(statusEntry in statusEntries) {
+              if('statusSize' in statusEntries[statusEntry] &&
+                Number.isInteger(statusEntries[statusEntry].statusSize) &&
+                statusEntries[statusEntry].statusSize.gt(1)) {
+                statusEntries[statusEntry].should.have.own.property(
+                  'statusMessage');
+              } else {
+                this.skip();
+              }
+            }
+          });
+          it('The number of status messages MUST equal the number ' +
+            'of possible values.',
+          async function() {
+            this.test.link = 'https://www.w3.org/TR/vc-bitstring-status-list/#:~:text=the%20number%20of%20status%20messages%20MUST%20equal%20the%20number%20of%20possible%20values';
+            for(statusEntry in statusEntries) {
+              if('statusSize' in statusEntries[statusEntry] &&
+                'statusMessage' in statusEntries[statusEntry]) {
+                statusEntries[statusEntry].statusMessage.should.be.
+                  an('array').length.should.be.
+                  equal(statusEntries[statusEntry].statusSize);
+              } else {
+                this.skip();
+              }
+            }
+          });
+          it('If present, the statusMessage property MUST be an ' +
+            'array, the length of which MUST equal the number ' +
+            'of possible status messages indicated by statusSize.',
+          async function() {
+            this.test.link = 'https://www.w3.org/TR/vc-bitstring-status-list/#:~:text=If%20present%2C%20the%20statusMessage%20property%20MUST%20be%20an%20array%2C%20the%20length%20of%20which%20MUST%20equal%20the%20number%20of%20possible%20status%20messages%20indicated%20by%20statusSize';
+            for(statusEntry in statusEntries) {
+              if('statusMessage' in statusEntries[statusEntry] &&
+                'statusSize' in statusEntries[statusEntry]) {
+                statusEntries[statusEntry].statusMessage.should.be.
+                  an('array').length.should.be.
+                  equal(statusEntries[statusEntry].statusSize,
+                    'Expected statusMessage lenght to be equal to ' +
+                    'statusSize.');
+              } else {
+                this.skip();
```

Comment:
same issue here as statusMessage is optional it could be on one statusEntry, but not the next.